spec/support/macros/define_constant.rb

```diff
@@ -28,9 +28,9 @@ def create_table(table_name, &block)
       connection.create_table(table_name, &block)
       @created_tables << table_name
       connection
-    rescue Exception => exception
+    rescue StandardError => error
```

Comment:
I think in this case rescuing`Exception` is appropriate because we are doing some cleanup and then re-raising the exception. If we use `StandardError` instead then terminating our program in the middle of this method might leave an unwanted table hanging around.

Reply:
I added a `rubocop:disable` comment for this instance, since I think it's still a good cop to have in most cases.